source/extensions/filters/http/jwt_authn/jwt_cache.cc

```diff
@@ -56,7 +60,10 @@ class JwtCacheImpl : public JwtCache {
   }
 
   void insert(const std::string& token, std::unique_ptr<::google::jwt_verify::Jwt>&& jwt) override {
-    if (jwt_lru_cache_ && token.size() <= kMaxJwtSizeForCache) {
+    if (!jwt_lru_cache_ || token.size() > std::numeric_limits<unsigned int>::max()) {
+      return;
+    }
+    if (static_cast<unsigned int>(token.size()) <= max_jwt_size_for_cache_) {
```

Comment:
This implies that only certain tokens will be cached (not a very intuitive cache use-case). Can you please clarify the comments in the API, and explicitly call that the max value is inclusive.

Reply:
Ok, added a comment to the proto file.